docs/tools/mdcode.py

```diff
@@ -27,13 +27,18 @@ def _parse_lines(self):
             m = _CODE_MARK.match(l)
             if m is not None:
                 if not in_code:
-                    lang = m.groups()[1].lower()
-                    indent = len(m.groups()[0])
-                yield (l, True, lang, indent)
-                if in_code:
+                    if _LANGS.has_key(m.groups()[1].lower()):
+                        lang = m.groups()[1].lower()
+                        indent = len(m.groups()[0])
+                        in_code = True
+                    elif m.groups()[1].lower() == 'bash':
+                        l = l.replace('bash', 'sh').lstrip()
```

Comment:
base is not in _LANGS, so it should never be converted as a notebook cell. also as i mentioned, a markdown tutorial should not have bash because it is not runnable in windows. so i suggest to remove this code block